src/compat.c

```diff
@@ -71,3 +72,28 @@ timescaledb_ddl_command_end(PG_FUNCTION_ARGS)
 	elog(ERROR, "Deprecated function should not be invoked");
 	PG_RETURN_NULL();
 }
+
+TS_FUNCTION_INFO_V1(invalidate_relcache_trigger);
```

Comment:
note: this will have to be removed when merging with the other PR (which removes compat.c).